Adding sovereign cloud authentication guidance (#778)
MicrosoftDocs/microsoftgraph-docs-powershell: microsoftgraph/docs-conceptual/authentication-commands.md

    1. For **Redirect URI**:
        - Select **Public client/native** from the drop down
        - **URI value**: *http://localhost*

>         - If you plan to use this application in a sovereign cloud environment, add an additional redirect URI with **Public client/native** and URI value: `ms-appx-web://Microsoft.AAD.BrokerPlugin/<YOUR_APP_CLIENT_ID>`. This URI is required for Windows Authentication Manager (WAM) broker-based sign-in in sovereign clouds.

Reviewer: No, this needs to be added in all cases!
```suggestion
        - An additional Redirect URI is required for Windows Authentication Manager (WAM) broker-based sign-in that will be added subsequently.
```

    1. Select **Register**.

>     1. Go to **Enterprise applications** and select the application you just created.

Reviewer: ```suggestion
    1. Click on **Authentication** in the left hand nav pane
    1. Click **+ Add Redirect URI**
    1. Click on **Mobile and desktop applications**
    1. Enter a URI value: `ms-appx-web://Microsoft.AAD.BrokerPlugin/<YOUR_APP_CLIENT_ID>` in the text box
    1. Click **Configure**
    1. Go to **Enterprise applications** and select the application you just created.
```
